CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Contributing to BaseApp
+
+Thank you for your interest in contributing to BaseApp! We welcome contributions from the community to help improve and grow the project. Please take a moment to review these guidelines before getting started.
+
+## Table of Contents
+
+- [Reporting Bugs](#reporting-bugs)
+- [Requesting Features](#requesting-features)
+- [Contributing Code](#contributing-code)
+- [Code of Conduct](#code-of-conduct)
+
+---
+
+## Reporting Bugs
+
+When you encounter a bug or unexpected behavior in BaseApp, please follow these steps to report it:
+
+1. Check the [GitHub Issues](https://github.com/silverlogic/baseapp-frontend/issues) to ensure that the bug hasn't already been reported.
+
+2. If the issue hasn't been reported, create a new issue by clicking on the "New Issue" button.
+
+3. Use a clear and descriptive title and provide detailed information about the issue, including steps to reproduce it and any error messages or logs.
+
+4. If possible, include any relevant code snippets or configuration files that can help in diagnosing the problem.
+
+5. Assign appropriate labels and milestones to the issue, if you have access.
+
+## Requesting Features
+
+If you have an idea for a new feature or enhancement in BaseApp, please follow these steps to request it:
+
+1. Check the [GitHub Issues](https://github.com/silverlogic/baseapp-frontend/issues) to ensure that the feature hasn't already been requested.
+
+2. If the feature hasn't been requested, create a new issue by clicking on the "New Issue" button.
+
+3. Use a clear and descriptive title and provide detailed information about the feature request, including its use case and benefits.
+
+4. If possible, include any design or implementation suggestions you may have.
+
+5. Assign appropriate labels and milestones to the issue, if you have access.
+
+## Contributing Code
+
+We welcome code contributions to BaseApp! To contribute code, please follow these steps:
+
+1. Fork the BaseApp repository to your GitHub account.
+
+2. Clone your forked repository to your local development environment.
+
+3. Create a new branch for your feature or bug fix:
+
+   ```
+   git checkout -b feature-name
+   ```
+
+4. Make your changes, write tests if applicable, and ensure that the existing tests pass.
+
+5. Commit your changes with a clear and concise commit message:
+
+   ```
+   git commit -m "Add feature: Your feature description"
+   ```
+
+6. Push your changes to your forked repository:
+
+   ```
+   git push origin feature-name
+   ```
+
+7. Open a Pull Request (PR) against the `master` branch of the BaseApp repository.
+
+8. Provide a descriptive PR title and include details about the changes you've made.
```

Comment:
maybe we could add another step to make sure people read the [versioning](https://github.com/silverlogic/baseapp-frontend/blob/master/README.md?plain=1#L96) section on the readme